tools/tidb-binlog-cluster.md

```diff
@@ -615,28 +615,30 @@ Command example:
 
 - Check the state of all the Pumps or Drainers:
 
+    Set `cmd` as `pumps` or `drainers` to check the state of all the Pumps or Drainers. For example:
+
     ```bash
-    bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps/drainers
+    bin/binlogctl -pd-urls=http://127.0.0.1:2379 -cmd pumps
 
-    2018/12/18 03:17:09 nodes.go:46: [info] pump: &{NodeID:1.1.1.1:8250 Addr:pump:8250 State:online IsAlive:false Score:0 Label:<nil> MaxCommitTS:405039487358599169 UpdateTS:405027205608112129}
+    INFO[0000] pump: {NodeID: ip-172-16-30-67:8250, Addr: 172.16.30.192:8250, State: online, MaxCommitTS: 405197570529820673, UpdateTime: 2018-12-25 14:23:37 +0800 CST}
     ```
 
-    > **Note:** Currently, the `IsAlive`, `Score` and `Label` fields are not used, so you do not need to pay attention to them.
-
 - Modify the Pump/Drainer state:
 
-    ```bash
-    The Pump/Drainer states include `online`, `pausing`, `paused`, `closing` and `offline`. 
+    Set `cmd` as `update-pump` or `update-drainer` to modify the Pump/Drainer state. The Pump/Drainer states include `online`, `pausing`, `paused`, `closing` and `offline`. 
```

Comment:
```suggestion
    Set `cmd` as `update-pump` or `update-drainer` to modify the state of Pump or Drainer, which can be `online`, `pausing`, `paused`, `closing` or `offline`.
```

Reply:
Generally, do not use `/` in the documentation body text, because it's a little bit informal.